addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/DataGenerator.java

```diff
@@ -173,6 +179,9 @@ public String generateValue(String name, Schema<?> schema, boolean isPath) {
     }
 
     private String getExampleValue(Parameter parameter) {
+        if (parameter.getSchema() == null) {
+            return "";
+        }
```

Comment:
This should no longer be needed.

Reply:
I think I got this correct now, I removed another redundant null check in `generateParam` and everything seems to be working correctly. Thanks for the 👀